usaspending_api/submissions/migrations/0011_submissionqueue.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2020-07-23 07:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0010_auto_20200623_1702'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SubmissionQueue',
+            fields=[
+                ('submission_id', models.IntegerField(primary_key=True, serialize=False)),
+                ('state', models.TextField(choices=[('NEW', 'NEW'), ('IN PROGRESS', 'IN PROGRESS'), ('FAILED', 'FAILED')], default='NEW')),
```

Comment:
In case you wanted the "new" state to be accurate after resets, it could be changed to "Ready" or "Queued"